Library/Homebrew/test/test_diagnostic.rb

```diff
@@ -141,4 +142,80 @@ def test_check_user_curlrc
         @checks.check_user_curlrc
     end
   end
+
+  def test_check_for_unsupported_curl_vars
+    MacOS.stubs(:version).returns :yosemite
```

Comment:
This needs to return either `OS::Mac::Version.new("10.10")` or `OS::Mac::Version.from_symbol(:yosemite)` for comparison to work.